fdbclient/RestoreWorkerInterface.actor.h

```diff
@@ -205,34 +246,28 @@ struct LoadingParam {
 	Key url;
 	Version prevVersion;
 	Version endVersion; // range file's mutations are all at the endVersion
-	int fileIndex;
-	std::string filename;
-	int64_t offset;
-	int64_t length;
+
 	int64_t blockSize;
```

Comment:
should `blockSize` be part of asset?

Reply:
Not really. The blockSize is an auxiliary information for restore loader to parse files, and it is not needed by appliers.
RestoreAsset is passed through both loaders and appliers. To save the message size, I consciously didn't include blockSize in RestoreAsset.